test/functional/apps/management/_getting_started.js

```diff
@@ -0,0 +1,59 @@
+import expect from 'expect.js';
+
+export default ({ getService, getPageObjects }) => {
+  const kibanaServer = getService('kibanaServer');
+  const esArchiver = getService('esArchiver');
+  const log = getService('log');
+
+  const PageObjects = getPageObjects(['common', 'gettingStarted']);
+
+  describe('Getting Started page', () => {
```

Comment:
Should we have a test that ensures the navigation is displayed properly when the user manually navigates to the Getting Starting page from the Management page?